Userland/Libraries/LibWeb/HTML/BrowsingContext.h

```diff
@@ -124,6 +124,10 @@ class BrowsingContext final
     HTML::WindowProxy* window_proxy();
     HTML::WindowProxy const* window_proxy() const;
 
+    JS::GCPtr<BrowsingContext> opener_browsing_context() const { return m_opener_browsing_context; }
+
+    void set_opener_browsing_context(BrowsingContext* browsing_context) { m_opener_browsing_context = browsing_context; }
```

Comment:
I'd argue we should use a `GCPtr` here as well :)

Reply:
Whoops, missed that one, fixed.